mteb/tasks/Classification/tur/TurkishProductSentimentClassification.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class TurkishProductSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="TurkishProductSentimentClassification",
+        description="Turkish Product Review Dataset",
+        reference="https://www.win.tue.nl/~mpechen/publications/pubs/MT_WISDOM2013.pdf",
+        dataset={
+            "path": "asparius/Turkish-Product-Review",
+            "revision": "ad861e463abda351ff65ca5ac0cc5985afe9eb99",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["tur-Latn"],
+        main_score="accuracy",
+        date=("2013-01-01", "2013-08-11"),
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="Not specified",
+        socioeconomic_status="mixed",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""
+        @inproceedings{Demirtas2013CrosslingualPD,
+            title={Cross-lingual polarity detection with machine translation},
+            author={Erkin Demirtas and Mykola Pechenizkiy},
+            booktitle={wisdom},
+            year={2013},
+            url={https://api.semanticscholar.org/CorpusID:3912960}
+        }
+        """,
+        n_samples={"train": 4800, "test": 1200},
```

Comment:
I think the test split is 800 samples, based on what I see on [HF](https://huggingface.co/datasets/asparius/Turkish-Product-Review/viewer/default/test)
```suggestion
        n_samples={"train": 4800, "test": 800},
```